admin/src/scenes/dashboardV2/components/Todos.jsx

```diff
@@ -21,11 +21,16 @@ export default function Todos({ user }) {
   const [fullNote, setFullNote] = useState(false);
   const [stats, setStats] = useState({});
   const cohorts = useSelector((state) => state.Cohorts);
-  const cohortsNotFinished = cohorts.filter((c) => new Date(c.dateEnd) > Date.now())?.map((e) => e.name);
+  const cohortsNotFinished = cohorts
+    .filter((c) => new Date(c.dateEnd) > Date.now())
+    ?.map((e) => e.name)
+    .filter((e) => !e.match(/CLE/));
```

Comment:
Je pense que tu peux te passer du `?`

```suggestion
  const cohortsNotFinished = cohorts
    .filter((c) => new Date(c.dateEnd) > Date.now())
    .map((e) => e.name)
    .filter((e) => !e.match(/CLE/));
```